Sloth.Web/Handlers/VerifyTeamPermissionHandler.cs

```diff
@@ -13,10 +15,14 @@ namespace Sloth.Web.Handlers
     public class VerifyTeamPermissionHandler : AuthorizationHandler<VerifyTeamPermission>
     {
         private readonly UserManager<User> _userManager;
+        private readonly IHttpContextAccessor _httpContext;
+        private readonly ITempDataDictionaryFactory _tempDataDictionaryFactory;
 
-        public VerifyTeamPermissionHandler(UserManager<User> userManager)
+        public VerifyTeamPermissionHandler(UserManager<User> userManager, IHttpContextAccessor httpContext, ITempDataDictionaryFactory tempDataDictionary)
         {
             _userManager = userManager;
+            _httpContext = httpContext;
+            _tempDataDictionaryFactory = tempDataDictionary;
```

Comment:
I don't see this being used anywhere?

Reply:
Oh, forgot to remove that. I think it was a holdover from AspNetCore 2.x that I had copypasta'd and then changed my mind. The code depending on it never seemed to get hit. I'll remove it...